_schemaV2.graphql

```diff
@@ -1841,9 +1841,6 @@ type Artist implements EntityWithFilterArtworksConnectionInterface & Node & Sear
     # The number of PartnerArtists to return
     size: Int
   ): [PartnerArtist]
-
-  # The Partner's provided biography for the artist
-  partnerBiographyBlurb(format: Format): partnerBiographyBlurb
```

Comment:
Doing a search returns a result in Eigen: https://github.com/search?q=partnerBiographyBlurb&type=code

I think it might be too late to remove this unfortunately, since a request in Eigen for this slice of the schema would break the now-released eigen version. 

So that's basically the flow. Do a global search, see if eigen is referencing it at all. If not, all good to remove. Otherwise we need to force a breaking change via force upgrade on mobile.

Reply:
Ah nice catch re: eigen
Ok cool - Maybe I'll move to deprecate and follow up after